Source/WebCore/page/LocalFrameView.h

```diff
@@ -89,6 +89,7 @@ class View;
 Pagination::Mode paginationModeForRenderStyle(const RenderStyle&);
 
 enum class LayoutViewportConstraint : bool { Unconstrained, ConstrainedToDocumentRect };
+enum class ScrollToTextFragmentIndicator : bool { Disabled, Enabled };
```

Comment:
I think this should be more like `ShowScrollToTextFragmentIndicator { No, Yes }`